de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java

```diff
@@ -251,4 +294,28 @@ public Response getConcepts(@DefaultValue("true") @QueryParam(QP_ONLY_ACTIVE_CON
 		}
 	
 	}
+	
+	/**
+	 * Create a new Sei typed by the Se identified by the fullQualifiedName
+	 * @param fullQualifiedName of the sei type (se)
+	 * @param owner of the sei (either another sei or the repository for root seis)
+	 * @param editingDomain
+	 * @return uuid of the created sei
+	 */
+	public static String createSeiFromFqn(String fullQualifiedName, EObject owner, VirSatTransactionalEditingDomain editingDomain) {
+		ActiveConceptHelper helper = new ActiveConceptHelper(editingDomain.getResourceSet().getRepository());
+		
+		StructuralElementInstance newSei = StructuralFactory.eINSTANCE.createStructuralElementInstance();

```

Comment:
StructuralInstantiator could be used here to simplify instantiation logic (creating sei + setting type + setting default name)